pyproject.toml

```diff
@@ -39,72 +43,66 @@ testing = [
 "Bug Reports" = "https://github.com/cfpb/wagtail-inventory/issues"
 "Source" = "https://github.com/cfpb/wagtail-inventory"
 
-[build-system]
-requires = ["setuptools>=43.0.0", "wheel"]
-build-backend = "setuptools.build_meta"
-
 [tool.setuptools.package-data]
 inventory = [
     "templates/wagtailinventory/*",
 ]
 
-[tool.black]
-line-length = 79
-target-version = ["py38"]
-include = '\.pyi?$'
-exclude = '''
-(
-  /(
-      \.eggs
-    | \.git
-    | \.tox
-    | \*.egg-info
-    | _build
-    | build
-    | dist
-    | migrations
-  )/
-)
-'''
-
-[tool.isort]
-profile = "black"
-line_length = 79
-lines_after_imports = 2
-skip = [".tox", "migrations", ".venv", "venv"]
-known_django = ["django"]
-known_wagtail = ["wagtail"]
-default_section = "THIRDPARTY"
-sections = [
-    "STDLIB",
-    "DJANGO",
-    "WAGTAIL",
-    "THIRDPARTY",
-    "FIRSTPARTY",
-    "LOCALFOLDER"
-]
+[tool.setuptools_scm]
 
 [tool.ruff]
+# Use PEP8 line-length
+line-length = 79
+# Exclude common paths
 exclude = [
     ".git",
     ".tox",
     "__pycache__",
-    "*/migrations/*.py",
-    "*/tests/testapp/migrations/*.py",
-    "docs",
-    ".venv",
-    "venv",
+    "**/migrations/*.py",
 ]
 
 [tool.ruff.lint]
-ignore = []
+ignore = ["E731", ]
+# Select specific rulesets to use
 select = [
+    # pycodestyle
     "E",
+    # pyflakes
     "F",
-    "W",
+    # flake8-bugbear
+    "B",
+    # pyupgrade
+    "UP",
+    # flake8-simplify
+    "SIM",
+    # isort
+    "I",
+]
+
+[tool.ruff.lint.isort.sections]
+"django" = ["django"]
+"wagtail" = ["wagtail"]
+
+[tool.ruff.lint.isort]
+lines-after-imports = 2
+known-third-party = ["jinja2"]
```

Comment:
This isn't needed here, is it? I think this comes from https://github.com/cfpb/consumerfinance.gov/commit/3ededa1a9d7a86e689173e444b0e0ec9e3365917, where we need it because we have a top-level cfgov/jinja2 path that would shadow imports from the third-party jinja2 package. Here we don't have that conflict so I don't think we need this exclusion.

Reply:
Removed in [67bf208](https://github.com/cfpb/wagtail-inventory/pull/83/commits/67bf208d55a89bd3395c02f17c3ef5fc9fc51db4)